constructor/header.sh

```diff
@@ -492,28 +492,42 @@ if [ "$BATCH" = "0" ]; then
        [ "$ans" != "y" ]   && [ "$ans" != "Y" ]
     then
         printf "\\n"
-        printf "You may wish to edit your .bashrc to prepend the __NAME__ install location to PATH:\\n"
+        printf "You may wish to edit your $BASH_RC to setup __NAME__:\\n"
         printf "\\n"
-        printf "export PATH=%s/bin:\$PATH\\n" "$PREFIX"
+        if [ -f "$PREFIX/etc/profile.d/conda.sh" ]; then
+            printf "source $PREFIX/etc/profile.d/conda.sh\\n"
+        else
+            printf "export PATH=\"$PREFIX/bin:\$PATH\"\\n"
+        fi
         printf "\\n"
     else
         if [ -f "$BASH_RC" ]; then
             printf "\\n"
-            printf "Appending source %s/bin/activate to %s\\n" "$PREFIX" "$BASH_RC"
+            printf "Initializing __NAME__ in %s\\n" "$BASH_RC"
             printf "A backup will be made to: %s-__name__.bak\\n" "$BASH_RC"
             printf "\\n"
             cp "$BASH_RC" "${BASH_RC}"-__name__.bak
         else
             printf "\\n"
-            printf "Appending source %s/bin/activate in\\n" "$PREFIX"
-            printf "newly created %s\\n" "$BASH_RC"
+            printf "Initializing __NAME__ in newly created %s\\n" "$BASH_RC"
         fi
+        cat <<EOF >> "$BASH_RC"
+# added by __NAME__ __VERSION__ installer
+# >>> conda initialize >>>
+# !! Contents within this block are managed by 'conda init' !!
+if [ -f "$PREFIX/etc/profile.d/conda.sh" ]; then
+    . "$PREFIX/etc/profile.d/conda.sh"
+else
+    export PATH="$PREFIX/bin:\$PATH"
+fi
+# <<< conda initialize <<<
+EOF
```

Comment:
We should go ahead and make this now what `conda init` will eventually do.  So the full block that `conda init` adds now, and actually exercise the first fallback until people get the new conda.

```
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="\$('$PREFIX/bin/conda' shell.bash hook 2> /dev/null)"
if [ \$? -eq 0 ]; then
    eval "\$__conda_setup"
else
    if [ -f "$PREFIX/etc/profile.d/conda.sh" ]; then
        . "$PREFIX/etc/profile.d/conda.sh"
    else
        export PATH="$PREFIX/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<
```

Reply:
I deliberately removed
```
'$PREFIX/bin/conda' shell.bash hook 2> /dev/null
```
Because it throws an exception at the moment and that makes opening a new shell incredibly slow.

Reply:
To be more specific, this throws a Not Implemented Exception.